mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -647,7 +647,9 @@ ::mediapipe::Status GlAnimationOverlayCalculator::GlSetup() {
     uniform sampler2D texture;  // texture to shade with
 
     void main() {
-      gl_FragColor = texture2D(texture, sample_coordinate);
+      vec4 pixel = texture2D(texture, sample_coordinate);
+      if (pixel.a < 0.2) discard;
```

Comment:
Can you make this a named constant and spend a couple of sentences on why you are doing this?